dev/gen_apidocs/create_documents.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Make Python documentation (Depends on C++ )
+cd /apache-arrow
+source activate pyarrow-dev
+rm -rf dist
+mkdir dist
+export ARROW_BUILD_TYPE=release
+export ARROW_HOME=$(pwd)/dist
+export PARQUET_HOME=$(pwd)/dist
+export LD_LIBRARY_PATH=$(pwd)/dist/lib:$LD_LIBRARY_PATH
+export BOOST_LIBRARYDIR=/opt/conda/envs/pyarrow-dev/lib
+export THRIFT_HOME=/opt/conda/envs/pyarrow-dev
```

Comment:
Should use `ARROW_BUILD_TOOLCHAIN`, `PARQUET_BUILD_TOOLCHAIN` if possible

Reply:
Could not get that to work. As some packages are in the OS location while others in Conda.